backend/coreapp/serializers.py

```diff
@@ -8,16 +8,17 @@
 def serialize_profile(request: Request, profile: Profile):
     if profile.user is None:
         return {
-            "is_you": profile == request.profile,
+            "is_you": profile == request.profile, # TODO(#245): remove
             "is_anonymous": True,
+            "id": profile.id,
         }
     else:
         user = profile.user
         github: Optional[GitHubUser] = GitHubUser.objects.filter(user=user).first()
         github_details = github.details() if github else None
 
         return {
-            "is_you": user == request.user,
+            "is_you": user == request.user, # TODO(#245): remove
```

Comment:
The frontend doesn't use is_you anymore so these can be removed assuming permuter doesn't use them

Reply:
It doesn't use "is_you", no